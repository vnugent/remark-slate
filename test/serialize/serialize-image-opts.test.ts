import { serialize, defaultNodeTypes } from '../../src';

it('Serialize an image from slate state to markdown', () => {
  expect(
    serialize({
      type: defaultNodeTypes.image,
      url: 'https://avatars.githubusercontent.com/u/2148168',
      caption: [
        { text: "Jack's profile picture with 'url' key instead of 'link" },
      ],
      children: [],
    })
  ).toMatchSnapshot();
});
