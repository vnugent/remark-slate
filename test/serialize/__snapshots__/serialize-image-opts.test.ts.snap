// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Serialize an image from slate state to markdown 1`] = `
"![Jack's profile picture with 'url' key instead of 'link](https://avatars.githubusercontent.com/u/2148168)
"
`;
